tests/storage/lvmconf_test.py

```diff
@@ -220,6 +223,26 @@ def test_real_lvm_conf():
         assert conf.getint("global", "use_lvmetad") in (1, 0, None)
 
 
+def test_configuted_value_devices_dir():
+    # Test lvmconf.configured_value() with value "devices/dir" which is very
+    # unlikely to be something else than /dev.
+    dev_dir = lvmconf.configured_value("devices/dir")
+    assert dev_dir == "/dev"
+
+
+def test_non_existing_config_value():
+    with pytest.raises(cmdutils.Error) as e:
+        lvmconf.configured_value("does/not/exists")
+    assert e.value.rc == 5
+
+
+def test_query_config_section():
+    with pytest.raises(se.UnexpectedLvmConfigOutput) as e:
+        val = lvmconf.configured_value("devices")
+        print("val=%s" % val)
```

Comment:
This print will never run and it it runs because the code did not raise, the outupt
is little bit confusing. Maybe:

   print(f"unexpected response: {val}")

Reply:
ops, this is forgotten debug print out, should be removed